index.js

```diff
@@ -91,6 +92,21 @@ module.exports = function(input, inputMap) {
 			});
 			return;
 		}
+
+		if (params.includeModulePaths) {
+			// Ensure module paths include full path from current working directory
+			var pwd = process.cwd();
+			if (context.substring(0, pwd.length) === pwd) {
```

Comment:
I'm not sure what you mean. Would the `if` be in the helper function? Is extracting ~10 lines of code to another function going to add clarity?